### DISCUSSION TITLE
After the update, Quick Launch turns off on Toolbar

### DISCUSSION THREAD

**Discussion by VV-SH** [valinet/ExplorerPatcher#425] [General]
Hi!
I'm talking about this:
![19-11-2021_12-03-53](https://user-images.githubusercontent.com/73989146/142595850-a36fa77e-e1af-4786-a6eb-6ff2c39d2906.png)

**Comment by valinet** [comment]
Have you read the release notes?

If not, then do.

**Reply by VV-SH** [reply]
Sorry! :(
Are you talking about this?
"Taskbar toolbar layouts are preserved when switching between Windows 10 and Windows 11 taskbars and in general (these will be reset when installing this update but should be subsequently remembered)"

**Reply by valinet** [reply]
Yup.
